src/shared/components/case-editor/case-edit-page/case-edit-page.html

```diff
@@ -10,7 +10,19 @@ <h1 class="govuk-heading-l">{{currentPage.label}}</h1>
 <h2 *ngIf="getCaseId()" class="heading-h2">
   #{{ getCaseId() | ccdCaseReference}}
 </h2>
-
+<!-- Error message summary -->
+<div *ngIf="validationErrors.length > 0" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+  <h2 class="govuk-error-summary__title" id="error-summary-title">
+    There is a problem
+  </h2>
+  <div *ngFor="let validationError of validationErrors" class="govuk-error-summary__body">
+    <ul class="govuk-list govuk-error-summary__list">
+      <li> 
+        <a (click)="navigateToErrorElement(validationError.id)" class="validationError">{{validationError.message}}</a>
```

Comment:
_Nitpick:_
CSS classes tend to be hyphenated rather than camel case: `validation-error`.

Reply:
updated